ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -200,12 +201,13 @@ - (void)logItemRetrievalStatus:(NSArray *)items
     if (!items || [items count]<=0)
     {
         //if resource is nil, this request is intending to find MRRT
-        AD_LOG_INFO(correlationId, @"No items were found for query: %@", keyCtxStr);
+        AD_LOG_INFO(correlationId, @"No items were found for query");
+        AD_LOG_INFO_PII(correlationId, @"No items were found for query %@", keyCtxStr);
     }
     else
     {
-        AD_LOG_INFO(correlationId, @"Found %lu token(s) for query: %@", (unsigned long)[items count], keyCtxStr);
-        AD_LOG_INFO(correlationId, @"userId: %@", userId);
+        AD_LOG_INFO(correlationId, @"Found %lu token(s) for query", (unsigned long)[items count]);
+        AD_LOG_INFO_PII(correlationId, @"Found %lu token(s) for query %@ user <%@>", (unsigned long)[items count], keyCtxStr, userId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 134 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)